de-lib/src/main/java/org/iplantc/de/intercom/client/IntercomFacade.java

```diff
@@ -0,0 +1,70 @@
+package org.iplantc.de.intercom.client;
+
+import org.iplantc.de.shared.DEProperties;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+/**
+ * A facade class that manages interaction with Intercom
+ * Created by sriram on 8/1/17.
+ */
+public class IntercomFacade {
```

Comment:
Can we make this class Injectable?  It seems most of the places we're using it are in the views, but if we end up needing this in a presenter, then the line where we add the tracking will need to be testable.  I think the current usage (IntercomFacade.trackEvent()) would cause Mockito to throw a NPE.  Also, then we could inject `DEProperties`.